Scripts/utils/print_links.py

```diff
@@ -1,6 +1,57 @@
+from typing import Iterable, Union, Tuple
+from shapely import Point, LineString
+
 from utils.calc_noise import NoiseModel
 
 
+class NODE:
+    geom_type = "Point"
+
+    def __new__(cls, node):
+        return Point(node.x, node.y)
+
+
+class LINK:
+    geom_type = "LineString"
+
+    def __new__(cls, link):
+        return LineString(link.shape)
+
+
+def geometries(scenario,
```

Comment:
scenario is of type indor.emme.database.scenario.Scenario that is dependent on Emme being available. I think this should be a Protocol (that is implemented by the Emme and Mock classes) or the function should be moved to the "emme only" parts of the code.